src/SprykerSdk/Spryk/SprykConfig.php

```diff
@@ -25,6 +25,10 @@ class SprykConfig
     protected const NAME_ORGANIZATION = 'spryker-sdk';
     protected const NAME_PACKAGE = 'spryk';
 
+    public const SPRYK_LEVEL_1 = 1;
+    public const SPRYK_LEVEL_2 = 2;
+    public const SPRYK_LEVEL_3 = 3;
```

Comment:
It seems that we're using only first one as public constant, what if we move them all into protected part, but instead create a new public one like `public const SPRYK_DEFAULT_DUMP_LEVEL = self::SPRYK_LEVEL_1;`. (not a change request, just thinking)

Reply:
fixed